File: templates/Uwp/Features/DeepLinking._VB/Activation/SchemeActivationConfig.vb
Namespace Activation
    Friend Module SchemeActivationConfig
        ' TODO WTS: Add the pages that can be opened from scheme activation in your app here.
        Private ReadOnly _activationPages As Dictionary(Of String, Type) = New Dictionary(Of String, Type)() From
        {
            {"sample", GetType(Views.SchemeActivationSamplePage)}
        }

        Friend Function GetPage(pageKey As String) As Type
            Return _activationPages.FirstOrDefault(Function(p) p.Key = pageKey).Value
        End Function

        Friend Function GetPageKey(pageType As Type) As String
            Return _activationPages.FirstOrDefault(Function(p) p.Value = pageType).Key
        End Function
    End Module
End Namespace

File: templates/Uwp/Features/DeepLinking/Activation/SchemeActivationConfig.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace Param_RootNamespace.Activation
{
    public static class SchemeActivationConfig
    {
        private static readonly Dictionary<string, Type> _activationPages = new Dictionary<string, Type>()
        {
            // TODO WTS: Add the pages that can be opened from scheme activation in your app here.
            { "sample", typeof(Views.SchemeActivationSamplePage) }
        };

        public static Type GetPage(string pageKey)
        {
            return _activationPages
                    .FirstOrDefault(p => p.Key == pageKey)
                    .Value;
        }

        public static string GetPageKey(Type pageType)
        {
            return _activationPages
                    .FirstOrDefault(v => v.Value == pageType)
                    .Key;
        }
    }
}
